BookHub-frontend/src/actions/alert_actions.js

```diff
@@ -0,0 +1,19 @@
+import { alertConstants } from '../_constants';
+ 
+export const alertActions = {
+    success,
+    error,
+    clear
+};
+ 
+function success(message) {
+    return { type: alertConstants.SUCCESS, message };
```

Comment:
action require dispatch function. 

this shoul look something like this 

```
return (dispatch) => {
			dispatch({
					payload: 'value',
					type: 'name'
				});
			})
	};
```

btw this actions is not used